lamindb/dev/datasets/__init__.py

```diff
@@ -22,13 +22,15 @@
    anndata_pbmc3k_processed
    anndata_with_obs
    anndata_suo22_Visium10X
+   anndata_file_pbmc68k_test
```

Comment:
Thanks for this! Could you list it next to `anndata_pbmc68k_reduced` and indicate the difference between these two assets? I'm still a bit confused why we need both in our "test dataset ontology" :D

Reply:
Yep, will do.